bigbluebutton-html5/private/locales/en.json

```diff
@@ -21,6 +21,7 @@
     "app.note.title": "Shared Notes",
     "app.note.label": "Note",
     "app.note.hideNoteLabel": "Hide note",
+    "app.note.tipLabel": "Press Esc to focus toolbar",
```

Comment:
Perhaps reword to give more context? Which toolbar - does the screen reader make it clear?